apps/condo/pages/_app.tsx

```diff
@@ -612,7 +614,23 @@ if (!isDisabledSsr || !isSSR()) {
 
             return { pageProps }
         } catch (error) {
-            console.error('Error while running `MyApp.getInitialProps', error)
+            const tooManyRequests = error?.cause?.result?.errors
+                ?.some((error) => error?.message?.toLowerCase().includes('many requests recently'))
```

Comment:
Can we use something other than a `message` to determine that this is a rate limit? Like code